validator/src/main/java/com/amazon/aoc/App.java

```diff
@@ -104,6 +106,21 @@ public class App implements Callable<Integer> {
       defaultValue = "true")
   private boolean isRollup;
 
+  @CommandLine.Option(
+      names = {"--kubeCfgFilePath"},
```

Comment:
super nit: all the rest of the parameters don't use camel case. can we keep the consistency?